packages/terra-date-input/src/DateInputUtil.js

```diff
@@ -245,5 +243,11 @@ DateInputUtil.inputType = {
   MONTH: 1,
   DAY: 2,
 };
+DateInputUtil.MinYearValue = 1900;
+DateInputUtil.MaxYearValue = 2100;
+DateInputUtil.keyCode = {
+  KEY_PLUS: 187,
```

Comment:
just so i understand, these keycodes aren't in the version of keycode js that we pull in?

Reply:
Yes

Reply:
Can we add a comment on here that they arn't included, seems a little odd if you didn't know that.

Reply:
Added comment c212e4e